docs/adding-a-new-xpi.md

```diff
@@ -26,6 +26,23 @@ git merge --allow-unrelated-histories template/master
 # fix conflicts, commit result
 ```
 
+### External repos
+
+What if I don't want to move my repository into mozilla-extensions?
+
+First, we need to make sure we meet the following requirements:
+
+- The repo needs to follow the security guidelines: branch protections, reviews, limit who is an admin. Private repos will need to invite the `mozilla-extensions/private-repo` team to enable automation.
+- The repo cannot be in the `mozilla` github organization.
+- The repo needs to be long-lived. If this addon is not going to be active in a year, we should follow the standard practices.
+
+Then we'll follow these steps:
+
+- Create a [ci-config patch like this](https://hg.mozilla.org/ci/ci-configuration/rev/232957b859b7078c6348e7c1004d8dac9111d8a7)
+- Enable the taskcluster integration once that patch is reviewed and landed
+- Inform SecOps of a new service, using [this form](https://github.com/mozilla-services/foxsec/issues/new?assignees=&labels=&template=01_NewService.md). If you don't have access, please contact SecOps in Slack `#secops-public`.
+- We'll still need to follow the rest of this doc, as well as the [Releasing a XPI](releasing-a-xpi.md) doc to fully set up the repository.
+
 ### Branch protection
 
 We will use the `master` branch as the main branch for releasing XPIs. It's important to set [branch protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-protected-branches) for the `master` branch, and get code review for the source. [SecOps](https://mana.mozilla.org/wiki/display/SVCOPS/Firefox+Operations+Security) will be auditing the repositories in the `mozilla-extensions` organization for compliance.
```

Comment:
Just happened to notice this specific reference to a mandated branch name.  I believe it may be out of date, as the "production" branch can be specified in the manifest files.

The SecOps audit code will use the production branch registered with us, or the "default branch" for the repository.